Gopkg.toml

```diff
@@ -118,23 +118,43 @@ required = [
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
 
 [[constraint]]
   name = "k8s.io/apiextensions-apiserver"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
 
 [[constraint]]
+  name = "k8s.io/client-go"
+  version = "kubernetes-1.15.4"
+
+[[override]]
```

Comment:
Without these overrides `dep ensure -update` falls into infinite loop or resolves wrong version (typically master).

Reply:
Ack thanks for the explanation. That matches the problems I had when I worked on this.